testeng/resources/cleanup-python-code-create-pr.sh

```diff
@@ -54,7 +54,11 @@ do_one_repo () {
   echo "Running script to create PR..."
   cd "$WORKSPACE/testeng-ci"
   pip install -r requirements/base.txt
-  local message="$(cat <<EOF
+  if [ -s "$repo_dir/.git/cleanup-python-code-message" ]
+  then
+     local message=$(cat "$repo_dir/.git/cleanup-python-code-message")
```

Comment:
Is there an explanation around the choice of dropping this file in `.git`?

Reply:
Seemed like a place unlikely to conflict with anything else

Reply:
Ok. Doesn’t feel right to me, but I don’t have a proposal, so I guess we’re going with it, unless you ask for other ideas. 🙂

Reply:
It's also a place that won't conflict with the rest of the working directory.